### DISCUSSION TITLE
release a musl install package?

### DISCUSSION THREAD

**Discussion by bitbegin** [helix-editor/helix#6528] [General]
The linux install package can't run on ubuntu18.04, so can release a musl version?

**Comment by David-Else** [comment]
Did you try https://docs.helix-editor.com/install.html#ubuntu ? It has been updated to Helix 23.03, but I don't know if it supports 18.04.

**Reply by bitbegin** [reply]
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found.

My working pc ubuntu can't access network, so i need a version build on ubuntu18.04.

**Comment by bitbegin** [comment]
musl version can run on some older systems.

**Reply by vmiheer** [reply]
The cluster only has rocky linux 8.8. I wish helix had musl build.

**Reply by pascalkuthe** [reply]
musl builds are only useful for portabiliby when musl is statically linked.

That is however not possible with helix because statically linking libc is incompatible with treesitter so this will never happen.

Its not hard to build helix yourself either in a docer container or on the cluster directly. We will/can only provide images build using the workers provided by github.
